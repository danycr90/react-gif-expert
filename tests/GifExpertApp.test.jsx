import { render, screen } from "@testing-library/react"
import { GifExpertApp } from '../src/GifExpertApp'

describe('Pruebas de <GifExpertApp />', () => {
     
  test('debe renderizar la pÃ¡gina', () => {
      
    const { container } = render(
      <GifExpertApp
        onNewCategory={ () => {} }   
      />
    )

    screen.debug()
    expect( container ).toMatchSnapshot();
  })

})