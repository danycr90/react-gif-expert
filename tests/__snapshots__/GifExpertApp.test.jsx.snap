// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas de <GifExpertApp /> debe de cambiar el valor de la caja de texto 1`] = `
<div>
  <h1>
    GifExpertApp
  </h1>
  <form
    aria-label="form"
  >
    <input
      placeholder="Buscar Gifs"
      type="text"
      value=""
    />
  </form>
  <h3>
    One Punch
  </h3>
  <h2>
    Cargando...
  </h2>
  <div
    class="card-grid"
  />
</div>
`;

exports[`Pruebas de <GifExpertApp /> debe renderizar la p√°gina 1`] = `
<div>
  <h1>
    GifExpertApp
  </h1>
  <form
    aria-label="form"
  >
    <input
      placeholder="Buscar Gifs"
      type="text"
      value=""
    />
  </form>
  <h3>
    One Punch
  </h3>
  <h2>
    Cargando...
  </h2>
  <div
    class="card-grid"
  />
</div>
`;
